Dockerfile.dev

```diff
@@ -0,0 +1,40 @@
+FROM ruby:2.5.8-alpine
+ENV LANG ja_JP.UTF-8
+ENV PAGER busybox less
+
+RUN apk update && \
+    apk upgrade && \
+    apk add --update\
+    bash \
+    build-base \
+    curl-dev \
+    git \
```

Comment:
Please insert [iproute2-minimal](https://pkgs.alpinelinux.org/package/edge/main/x86_64/iproute2-minimal).

Reply:
Thanks for reviewing. Added!